sdk/fixtures/src/main/java/com/klaviyo/fixtures/BaseTest.kt

```diff
@@ -80,10 +109,22 @@ abstract class BaseTest {
         every { networkJitterRange } returns 0..0
         every { baseUrl } returns "https://test.fake-klaviyo.com"
         every { apiRevision } returns "1234-56-78"
+        every { baseCdnUrl } returns "https://decent.cdn.url.com"
+        every { assetSource } returns null
+        every { sdkName } returns "klaviyo-android-sdk"
+        every { sdkVersion } returns "4.20.69"
+    }
+    protected val mockContentView: ViewGroup = mockk(relaxed = true)
+    protected val mockDecorView: View = mockk(relaxed = true) {
+        every { findViewById<ViewGroup>(any()) } returns mockContentView
+    }
+    protected val mockActivity: Activity = mockk(relaxed = true) {
+        every { window.decorView } returns mockDecorView
```

Comment:
Same with these props, since they're only used in one test file, move them to that file plz